src/chiron_utils/bots/__init__.py

```diff
@@ -31,6 +34,7 @@
         [
             LrAdvisor,
             LrPlayer,
+            ElasticAdvisor,
```

Comment:
This list should be sorted:

```suggestion
            ElasticAdvisor,
            LrAdvisor,
            LrPlayer,
```